docs/components/Chips/Web.stories.tsx

```diff
@@ -26,27 +24,21 @@ export default {
 
 const BasicTemplate: ComponentStory<typeof Chips> = args => {
   const [selected, setSelected] = useState<string>();
+
   return (
     <Content>
       <Text>
         You are <u>{selected ? selected : "_______"}</u>
       </Text>
-      <Chips {...args} selected={selected} onChange={setSelected}>
-        <Chip
-          prefix={<Avatar initials="AZ" />}
-          label="Amazing"
-          value="Amazing"
-        />
-        <Chip
-          prefix={<Icon name="video" />}
-          label="Wonderful"
-          value="Wonderful"
-        />
-        <Chip
-          prefix={<Icon name="starFill" />}
-          label="Brilliant"
-          value="Brilliant"
-        />
+      <Chips
```

Comment:
New design for this doesn't have the prefixes, so I've removed them for the test.